pandas/tests/indexes/test_multi.py

```diff
@@ -632,6 +632,13 @@ def test_from_arrays_index_series_period(self):
 
         tm.assert_index_equal(result, result2)
 
+    def test_from_arrays_different_lengths(self):
+        # GH13599
+        idx1 = [1, 2, 3]
```

Comment:
you simply construct an expected MultiIndex and use ``.equals`` that it is equal to a result that uses ``.from_arrays``. I am sure there are other examples to model from.